<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentLogin</class>
 <widget class="QWidget" name="StudentLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>学生管理系统</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="indentation">
      <number>8</number>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>学生管理系统</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>学生管理系统</string>
      </property>
      <item>
       <property name="text">
        <string>学生成绩查询</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>学生信息查询</string>
       </property>
      </item>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>200</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="stuexam">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QComboBox" name="cbb_year">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QComboBox" name="cbb_mou">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>期中</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>期末</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QComboBox" name="cbb_term">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>上</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>下</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="9" colspan="2">
        <widget class="QPushButton" name="btn_search">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>搜索</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" colspan="2">
        <widget class="QLabel" name="label_year">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>学年</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="label_term">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>学期</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_number">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>第</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="13">
        <widget class="QTableWidget" name="tableWidget">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::TargetMoveAction</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectColumns</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideMiddle</enum>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>学年</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string>语文</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>数学</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>英语</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>政治</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>历史</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>地理</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>生物</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>物理</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>化学</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>总分</string>
          </property>
         </row>
        </widget>
       </item>
       <item row="0" column="11">
        <widget class="QPushButton" name="btn_cancel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stuinfo">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="5" column="4">
        <widget class="QLabel" name="lb_tel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>家长电话：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5" colspan="2">
        <widget class="QLineEdit" name="le_age">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QLabel" name="lb_passward">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="le_account">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QLabel" name="lb_class">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>班级：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="lb_age">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>年龄：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5" colspan="2">
        <widget class="QLineEdit" name="le_id">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="5" colspan="2">
        <widget class="QLineEdit" name="le_tel">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="9">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="le_grade">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lb_name">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>姓名：</string>
         </property>
        </widget>
       </item>
       <item row="6" column="5" colspan="2">
        <widget class="QLineEdit" name="le_passward">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="le_name">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="lb_id">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>学号：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="le_sex">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="le_session">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="lb_account">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>账号：</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="lb_session">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>届：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="lb_sex">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>性别：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="4">
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>学生信息</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="lb_grade">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>年级：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5" colspan="2">
        <widget class="QLineEdit" name="le_class">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="10">
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>720</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_need_upd">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>我要修改</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_update">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>确认修改</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>137</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="btn_exit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="lb_info">
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>学号：
姓名：</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>treeWidget</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>horizontalSpacer_7</zorder>
  <zorder>btn_exit</zorder>
  <zorder>lb_info</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
